llvm/test/tools/llvm-profdata/memprof-yaml.test

```diff
@@ -1,9 +1,11 @@
-; REQUIRES: x86_64-linux
 ; RUN: split-file %s %t
 ; RUN: llvm-profdata merge %t/memprof-in.yaml -o %t/memprof-out.indexed
 ; RUN: llvm-profdata show --memory %t/memprof-out.indexed > %t/memprof-out.yaml
 ; RUN: cmp %t/memprof-in.yaml %t/memprof-out.yaml
 
+; This test is expected to fail until the profile format is updated to handle CalleeGuids.
```

Comment:
Is the other option to just revert (some of?) the changes to this test?

Reply:
Unfortunately no since we changed the type from vector<vector<Frame>> to vector<struct>. I could introduce a sibling vector for the CalleeGuids instead where the same index in each vector correspond to the same callsite but that seems less readable. Wdyt?

Reply:
Resolving.